client/my-sites/importer/importer-header.jsx

```diff
@@ -140,4 +154,7 @@ const mapDispatchToProps = dispatch => ( {
 	),
 } );
 
-export default connectDispatcher( null, mapDispatchToProps )( localize( ImporterHeader ) );
+export default connect(
+	null,
+	{ recordTracksEvent }
```

Comment:
Can the `recordTracksEvent` here and blow not go in the respective `mapDispatchToProps` return values?

It seems odd to have both `connect` & `connectDispatcher` in play.

Reply:
As mentioned [earlier](https://github.com/Automattic/wp-calypso/pull/25120#discussion_r197494812), this is due to the Import section using a Flux store that's modelled after how Redux works. Since `connectDispatcher` talks to the Flux store, I think we can't have the `recordTracksEvent` exposed there, as it's Redux action.